contrib/face_recognition.lua

```diff
@@ -303,6 +306,9 @@ local function face_recognition ()
                 t = nonpersonsfoundTag
               end
               if t ~= "" and t ~= nil then
+                if categoryTagString ~= "" then
+                  t = categoryTagString .. "|" .. t
```

Comment:
This prepends the category string to all tags including no_persons_found and unknown_person.  While People|unknown_person makes a kind of sense, People:no_persons_found doesn't.  I suggest just adding the category tag before the recognized persons only.